src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -757,7 +757,9 @@ static unique_ptr<Scene> _sceneBuildHelper(const SvgNode* node, const Box& vBox,
 static void _applySvgViewFlag(const Scene* scene, float& vx, float& vy, float& vw, float& vh, float& w, float& h, SvgViewFlag viewFlag)
 {
     if (!((uint32_t)viewFlag & (uint32_t)SvgViewFlag::Viewbox)) {
-        scene->bounds(nullptr, nullptr, &vw, &vh, false);
+        scene->bounds(&vx, &vy, &vw, &vh, false);
```

Comment:
This looks leading counter dependency... bounds() depends on svg loader result.

Reply:
@mgrudzinska Thank you for fixing it. From a design point of view, calling scene->bounds() is not ideal for the reasons mentioned above. I have reviewed it twice, nonetheless, I believe that applying original patch is better since we now understand what we are doing. What are your thoughts? You can leave this patch for the future refactoring. Apologies for my previous hasty comment.

Reply:
@hermet no problem, there were so many things to think of in this 'no viewbox and width/height' issue, that as tthis point I was confused what was applied, what wasn;t... sorry for that, thx for your all revs.
I separated the first commit from this pr, so we can aplly it and I'll close this one - I don;t like hanging prs, if I'll need this one I can find it in closed once.